a3p-integration/proposals/z:acceptance/README.md

```diff
@@ -0,0 +1,6 @@
+# Inert proposal to test acceptance of the history of upgrades
+
+It's `type` is `/cosmos.params.v1beta1.ParameterChangeProposal` because that is
```

Comment:
```suggestion
Its `type` is `/cosmos.params.v1beta1.ParameterChangeProposal` because that is
```

Reply:
Other than that do you Approve? It would save time to say that you approve it conditional on some requests and rely on the author to make them before merging.